server/form-pages/apply/area-and-funding/funding-information/fundingInformation.ts

```diff
@@ -39,10 +35,7 @@ export default class FundingSource implements TaskListPage {
     this.body = body as FundingSourceBody
 
     const applicationQuestions = getQuestions(this.personName)
-    this.questions = {
-      fundingSource: applicationQuestions['funding-information']['funding-source'].fundingSource.question,
-    }
-    this.options = applicationQuestions['funding-information']['funding-source'].fundingSource.answers
```

Comment:
Is this remodelling all happening because we don't have to present the 'both' option?